deploy/common/env.sh

```diff
@@ -66,6 +66,8 @@ echo "ROX_HOTRELOAD is set to ${ROX_HOTRELOAD}"
 export ROX_DECLARATIVE_CONFIGURATION="${ROX_DECLARATIVE_CONFIGURATION:-}"
 echo "ROX_DECLARATIVE_CONFIGURATION is set to ${ROX_DECLARATIVE_CONFIGURATION}"
 
+echo "ROX_CENTRAL_NOTIFICATIONS is set to ${ROX_CENTRAL_NOTIFICATIONS}"
```

Comment:
I think it would be great to stick to a single name of "the thing" in schema, service, env var, etc. (In previous PR "the thing" is called just "notifications".)